packages/web-components/@groupby/elements-search-box/stories/index.ts

```diff
@@ -1,7 +1,13 @@
 import { storiesOf } from '@storybook/html';
 import { withKnobs, text, boolean } from '@storybook/addon-knobs';
-import { UPDATE_SEARCH_TERM } from '@groupby/elements-events';
-import { getDisplayCode } from '../../../../../.storybook/common';
+import {
+  SEARCHBOX_CLEAR,
+  SEARCHBOX_CLICK,
+  SEARCHBOX_INPUT,
+  SEARCH_REQUEST,
+  UPDATE_SEARCH_TERM
```

Comment:
Please use trailing commas. 
```suggestion
  UPDATE_SEARCH_TERM,
```